packages/amplify-graphql-api-construct/src/__tests__/__functional__/resources.test.ts

```diff
@@ -347,6 +347,124 @@ describe('generated resource access', () => {
       });
     });
 
+    describe('default vs amplify-managed tables', () => {
```

Comment:
I added this while troubleshooting an unrelated issue because I didn't see tests that asserted the resources created for each dynamo strategy. If such a test already exists, we can remove this since we don't particularly want redundant tests.